commit 5f98f99744f0ec1945ad541ad5ad6b28c09f00b8 (refs/changes/02/1902/3)
Author: Jason King <jason.king@joyent.com>
Date:   2017-05-04T14:06:20-05:00 (2 years, 5 months ago)
    
    NAPI-412 portolan-moray.logReq should sort results
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/consumer.js b/lib/consumer.js
index f135a09..99e38cc 100644
--- a/lib/consumer.js
+++ b/lib/consumer.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2016, Joyent, Inc.
+ * Copyright 2017, Joyent, Inc.
  */
 
 /*
@@ -304,7 +304,12 @@ function logReq(opts, callback) {
 
     var cnUuid = opts.cnUuid;
     var filter = fmt('cn_uuid=%s', cnUuid);
-    var searchOpts = { limit: opts.limit };
+    var searchOpts = {
+        limit: opts.limit,
+        sort: {
+            attribute: '_mtime'
+        }
+    };
     var logs = [];
 
     /*
